integration_tests/test_issue_518.py

```diff
@@ -0,0 +1,14 @@
+from ltypes import i64
+
+def fib(n: i64) -> i64:
+    if n < 2:
+        return n
+    else:
+        return fib(n - 1) + fib(n - 2)
+
+def main():
+    ans: i64
+    ans = fib(35)
```

Comment:
This makes it run "immediately":
```suggestion
    ans = fib(15)
```
The previous number was fast with LPython, but slow with CPython.